src/components/AdminNavbar/AdminNavbar.tsx

```diff
@@ -1,25 +1,41 @@
 import React from 'react';
 import styles from './AdminNavbar.module.css';
+import { Nav } from 'react-bootstrap';
+import Navbar from 'react-bootstrap/Navbar';
 import Logo from 'assets/talawa-logo-lite-200x200.png';
-
 interface NavbarProps {
   targets: { url: string; name: string }[];
 }
 
 function AdminNavbar({ targets }: NavbarProps): JSX.Element {
```

Comment:
Sir the margin is due to the page that is being made not becsur of navbar.

As soon as the working on page will start it will be good, the navbar and sidebar have got nothing to do with that.